core/src/main/java/org/pegdown/ParserWithDirectives.java

```diff
@@ -162,7 +162,7 @@ public Node parseBlockContents(Var<Integer> markerLength, StringBuilderVar block
         // recursively parse inner contents
         Context<Object> context = getContext();
         Integer length = markerLength.get();
-        String contents = block.getString();
+        String contents = "\n" + block.getString() + "\n";
```

Comment:
I found this behaviour surprising when I used the callouts in Akka HTTP which is why I reported it as a bug and made a PR. Since Paradox already appends `\n\n` to pages and appends `\n` to block content, having it instead append `\n\n` to block content (in the updated patch) would IMO make things more consistent.

Why should content inside block directives render different as if it was put in a standalone page?

The current behaviour is:
```
@@@ note
A

B
@@@
```
renders as
```
<div ...>
<div ...>title</div>
<p>A</A>
B
</div>
```